{"level":"error","message":"Failed to start server: generalRateLimit is not defined","service":"secure-auth-api","stack":"ReferenceError: generalRateLimit is not defined\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:43:13)","timestamp":"2025-09-11T09:24:04.688Z"}
{"level":"error","message":"Failed to start server: generalRateLimit is not defined","service":"secure-auth-api","stack":"ReferenceError: generalRateLimit is not defined\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:44:13)","timestamp":"2025-09-11T09:24:44.418Z"}
{"level":"error","message":"Failed to start server: generalRateLimit is not defined","service":"secure-auth-api","stack":"ReferenceError: generalRateLimit is not defined\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:44:13)","timestamp":"2025-09-11T09:27:29.393Z"}
{"level":"error","message":"Failed to start server: generalRateLimit is not defined","service":"secure-auth-api","stack":"ReferenceError: generalRateLimit is not defined\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:44:13)","timestamp":"2025-09-11T09:30:01.345Z"}
{"level":"error","message":"Failed to start server: generalRateLimit is not defined","service":"secure-auth-api","stack":"ReferenceError: generalRateLimit is not defined\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:44:13)","timestamp":"2025-09-11T09:30:31.556Z"}
{"level":"error","message":"Failed to start server: app.use() requires a middleware function","service":"secure-auth-api","stack":"TypeError: app.use() requires a middleware function\n    at Function.use (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\express@5.1.0\\node_modules\\express\\lib\\application.js:213:11)\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:45:9)","timestamp":"2025-09-11T09:30:59.679Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/invalid-phone-number","message":"The phone number must be a non-empty E.164 standard compliant identifier string."},"level":"error","message":"Registration error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at validateCreateEditRequest (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:357:15)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:650:5\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1601:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:29:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:51:22)","timestamp":"2025-09-11T09:48:54.954Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:78:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:51:22)","timestamp":"2025-09-11T09:48:54.956Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/invalid-phone-number","message":"The phone number must be a non-empty E.164 standard compliant identifier string."},"level":"error","message":"Registration error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at validateCreateEditRequest (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:357:15)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:650:5\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1601:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:29:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:51:22)","timestamp":"2025-09-11T09:55:59.656Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:78:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:51:22)","timestamp":"2025-09-11T09:55:59.658Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/invalid-phone-number","message":"The phone number must be a non-empty E.164 standard compliant identifier string."},"level":"error","message":"Registration error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at validateCreateEditRequest (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:357:15)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:650:5\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1601:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:29:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T09:57:38.094Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:78:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T09:57:38.097Z"}
{"level":"error","message":"Registration error: import_redis.redisClient.setex is not a function","service":"secure-auth-api","stack":"TypeError: import_redis.redisClient.setex is not a function\n    at AuthService.generateTokens (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:298:23)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:62:22)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:05:48.034Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:78:13)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:05:48.037Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/email-already-exists","message":"The email address is already in use by another account."},"level":"error","message":"Registration error: The email address is already in use by another account.","service":"secure-auth-api","stack":"Error: The email address is already in use by another account.\n    at FirebaseAuthError.fromServerError (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\utils\\error.js:148:16)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1629:49\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:12:53.680Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:12:53.682Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/invalid-phone-number","message":"The phone number must be a non-empty E.164 standard compliant identifier string."},"level":"error","message":"Registration error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at validateCreateEditRequest (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:357:15)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:650:5\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1601:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:19:06.857Z"}
{"level":"error","message":"Registration controller error: Registration failed","service":"secure-auth-api","stack":"Error: Registration failed\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:19:06.860Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/invalid-phone-number","message":"The phone number must be a non-empty E.164 standard compliant identifier string."},"level":"error","message":"Registration error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at validateCreateEditRequest (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:357:15)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:650:5\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1601:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:21:54.481Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: The phone number must be a non-empty E.164 standard compliant identifier string.","service":"secure-auth-api","stack":"Error: Registration failed: Error: The phone number must be a non-empty E.164 standard compliant identifier string.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-11T10:21:54.483Z"}
{"level":"error","message":"Failed to create audit log: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\collection-reference.js:214:52\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)\n    at EnabledTraceUtil.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:102:28)","timestamp":"2025-09-11T10:22:24.988Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::3001","port":3001,"service":"secure-auth-api","stack":"Error: listen EADDRINUSE: address already in use :::3001\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\express@4.21.2\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:69:9)","syscall":"listen","timestamp":"2025-09-11T14:44:01.711Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::3001","port":3001,"service":"secure-auth-api","stack":"Error: listen EADDRINUSE: address already in use :::3001\n    at Server.setupListenHandle [as _listen2] (node:net:1939:16)\n    at listenInCluster (node:net:1996:12)\n    at Server.listen (node:net:2101:7)\n    at Function.listen (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\express@4.21.2\\node_modules\\express\\lib\\application.js:635:24)\n    at startServer (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\index.ts:69:9)","syscall":"listen","timestamp":"2025-09-11T14:47:32.171Z"}
{"level":"error","message":"Failed to create audit log: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\collection-reference.js:214:52\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)\n    at EnabledTraceUtil.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:102:28)","timestamp":"2025-09-11T14:49:55.212Z"}
{"level":"error","message":"Failed to create audit log: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\collection-reference.js:214:52\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)\n    at EnabledTraceUtil.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:102:28)","timestamp":"2025-09-11T14:50:27.092Z"}
{"level":"error","message":"Token refresh error: Invalid refresh token","service":"secure-auth-api","stack":"Error: Invalid refresh token\n    at AuthService.refreshTokens (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:166:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async refreshToken (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:126:22)","timestamp":"2025-09-11T14:50:27.144Z"}
{"level":"error","message":"Token refresh controller error: Invalid refresh token","service":"secure-auth-api","stack":"Error: Invalid refresh token\n    at AuthService.refreshTokens (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:201:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async refreshToken (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:126:22)","timestamp":"2025-09-11T14:50:27.150Z"}
{"level":"error","message":"Failed to create audit log: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\collection-reference.js:214:52\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)\n    at EnabledTraceUtil.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:102:28)","timestamp":"2025-09-11T14:50:50.948Z"}
{"code":"ECONNRESET","errno":-4077,"level":"error","message":"Redis Client Error: read ECONNRESET","service":"secure-auth-api","stack":"Error: read ECONNRESET\n    at TCP.onStreamRead (node:internal/stream_base_commons:216:20)","syscall":"read","timestamp":"2025-09-13T09:23:03.532Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:17.567Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:17.810Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:18.022Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:18.272Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:18.814Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:19.712Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:21.330Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:23.460Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:25.564Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:27.597Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:29.766Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:31.787Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:33.978Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:36.009Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:38.056Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:40.271Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:42.455Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:44.603Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:46.747Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:48.951Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:50.979Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:53.178Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:55.233Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:57.326Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:51:59.383Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:01.582Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:03.686Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:05.721Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:07.871Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:09.930Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:12.041Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:14.076Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:16.277Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:18.345Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:20.546Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:22.581Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:24.631Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:26.832Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:28.899Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:30.946Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:33.016Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:35.055Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:37.129Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:39.186Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:41.227Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:43.294Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:45.364Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:47.546Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:49.670Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:51.857Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:54.028Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:56.191Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:52:58.365Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:00.506Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:02.526Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:04.707Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:06.870Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:18.004Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:18.327Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:18.646Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:18.998Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:19.629Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:20.486Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:22.125Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:40.501Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:40.679Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:40.883Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:41.160Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:41.786Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:42.715Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:44.363Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:46.507Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:48.540Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:53:50.776Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:46.151Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:46.417Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:46.594Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:46.894Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:47.390Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:48.298Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:50.009Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:52.171Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:54.262Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:56.292Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:55:58.511Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:10.748Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:11.015Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:11.299Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:11.527Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:12.032Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:12.954Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:14.727Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:19.084Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:19.343Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:19.506Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:19.733Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:20.334Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:21.154Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:22.875Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:25.082Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:27.198Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:29.701Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:32.431Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:34.492Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:36.619Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:38.730Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:40.757Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:42.827Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:44.894Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:47.087Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:49.282Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:51.352Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:53.409Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:55.602Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:56:57.791Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:00.002Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:02.035Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:04.210Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:06.394Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:08.468Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:10.515Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:12.634Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:14.700Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:16.838Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:19.047Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:21.205Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:23.397Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:25.525Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:27.594Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:29.805Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:31.930Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:34.048Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:36.236Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:38.422Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:40.511Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:42.645Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:44.743Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:46.826Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:48.967Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:51.091Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:53.207Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:55.300Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:57.478Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:57:59.615Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:01.689Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:03.776Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:05.857Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:07.916Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:10.003Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:12.076Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:14.281Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:16.398Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:18.631Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:20.663Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:22.702Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:24.812Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:26.958Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:29.013Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:31.199Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:33.379Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:35.541Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:37.563Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:39.763Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:41.945Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:44.023Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:46.225Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:48.274Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:50.326Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:52.433Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:54.574Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:56.646Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:58:58.699Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:00.831Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:02.977Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:05.024Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:07.180Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:09.312Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:11.638Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:13.876Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:16.052Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:18.266Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:20.314Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:22.527Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:24.714Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:26.904Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:28.968Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:31.130Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:33.310Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:35.399Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:37.525Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:39.738Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:41.938Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:44.063Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:46.180Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:48.385Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:50.567Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:52.634Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:54.704Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:56.781Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T05:59:58.995Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:01.169Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:03.304Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:05.485Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:07.623Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:09.725Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:11.767Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:13.863Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:15.999Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:18.148Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:20.369Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:22.480Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:24.562Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:26.737Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:28.835Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:30.928Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:33.088Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:35.158Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:37.244Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:39.259Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:00:41.420Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:02.726Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:03.005Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:03.208Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:03.526Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:03.976Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:04.830Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:06.538Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:08.681Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:10.748Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:12.838Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:14.915Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:17.025Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:19.041Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:21.132Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:23.268Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:25.445Z"}
{"code":"ECONNREFUSED","level":"error","message":"Redis Client Error:","service":"secure-auth-api","stack":"AggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1139:18)\n    at afterConnectMultiple (node:net:1714:7)","timestamp":"2025-09-14T06:01:27.594Z"}
{"level":"error","message":"Registration error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at WriteBatch.set (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:263:9)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\document-reference.js:348:41\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)","timestamp":"2025-09-14T06:52:35.729Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Registration failed: Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:52:35.749Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/email-already-exists","message":"The email address is already in use by another account."},"level":"error","message":"Registration error: The email address is already in use by another account.","service":"secure-auth-api","stack":"Error: The email address is already in use by another account.\n    at FirebaseAuthError.fromServerError (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\utils\\error.js:148:16)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1629:49\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:53:55.707Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: The email address is already in use by another account.","service":"secure-auth-api","stack":"Error: Registration failed: Error: The email address is already in use by another account.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:53:55.709Z"}
{"level":"error","message":"Registration error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at WriteBatch.set (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:263:9)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\document-reference.js:348:41\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)","timestamp":"2025-09-14T06:54:05.769Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Registration failed: Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"displayName\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:54:05.772Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/email-already-exists","message":"The email address is already in use by another account."},"level":"error","message":"Registration error: The email address is already in use by another account.","service":"secure-auth-api","stack":"Error: The email address is already in use by another account.\n    at FirebaseAuthError.fromServerError (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\utils\\error.js:148:16)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1629:49\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:54:35.996Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: The email address is already in use by another account.","service":"secure-auth-api","stack":"Error: Registration failed: Error: The email address is already in use by another account.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:54:36.003Z"}
{"codePrefix":"auth","errorInfo":{"code":"auth/phone-number-already-exists","message":"The user with the provided phone number already exists."},"level":"error","message":"Registration error: The user with the provided phone number already exists.","service":"secure-auth-api","stack":"Error: The user with the provided phone number already exists.\n    at FirebaseAuthError.fromServerError (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\utils\\error.js:148:16)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\firebase-admin@13.5.0\\node_modules\\firebase-admin\\lib\\auth\\auth-api-request.js:1629:49\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:33:26)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:54:43.027Z"}
{"level":"error","message":"Registration controller error: Registration failed: Error: The user with the provided phone number already exists.","service":"secure-auth-api","stack":"Error: Registration failed: Error: The user with the provided phone number already exists.\n    at AuthService.register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\services\\auth.service.ts:84:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async register (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\src\\controllers\\auth.controller.ts:52:22)","timestamp":"2025-09-14T06:54:43.033Z"}
{"level":"error","message":"Failed to create audit log: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.","service":"secure-auth-api","stack":"Error: Value for argument \"data\" is not a valid Firestore document. Cannot use \"undefined\" as a Firestore value (found in field \"resourceId\"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:383:19)\n    at validateUserInput (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\serializer.js:375:13)\n    at validateDocumentData (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\write-batch.js:609:40)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\reference\\collection-reference.js:214:52\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:110:30\n    at NoopContextManager.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\context\\NoopContextManager.ts:31:15)\n    at ContextAPI.with (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\api\\context.ts:77:42)\n    at NoopTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\NoopTracer.ts:98:27)\n    at ProxyTracer.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@opentelemetry+api@1.9.0\\node_modules\\@opentelemetry\\api\\src\\trace\\ProxyTracer.ts:51:20)\n    at EnabledTraceUtil.startActiveSpan (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\@google-cloud+firestore@7.11.3\\node_modules\\@google-cloud\\firestore\\build\\src\\telemetry\\enabled-trace-util.js:102:28)","timestamp":"2025-09-14T06:54:56.921Z"}
{"error":"Expected double-quoted property name in JSON at position 65 (line 4 column 1)","ip":"::1","level":"error","message":"Unhandled error:","method":"POST","service":"secure-auth-api","stack":"SyntaxError: Expected double-quoted property name in JSON at position 65 (line 4 column 1)\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\body-parser@1.20.3\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\body-parser@1.20.3\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:214:14)\n    at invokeCallback (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\raw-body@2.5.2\\node_modules\\raw-body\\index.js:238:16)\n    at done (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\raw-body@2.5.2\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (C:\\Users\\ADMIN\\Documents\\PNPM\\nodejs-nextjs-firebase\\server\\node_modules\\.pnpm\\raw-body@2.5.2\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:518:28)\n    at endReadableNT (node:internal/streams/readable:1698:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-09-14T07:13:57.397Z","url":"/api/user","userAgent":"PostmanRuntime/7.46.0"}
